.gitignore

```diff
@@ -12,4 +12,5 @@ testconfig.json
 /package.json
 
 // Private Tarballs
-*.tgz
\ No newline at end of file
+*.tgz
+*.snap
```

Comment:
I see -- I assumed these had something to do with the debugging I was doing and VS Code related files.. I will remove the .gitignore.